### PR TITLE
WIP EasyBuild recipe for Cosmo Pompa 24d1ac6 + dependencies cdt/20.08

### PR DESCRIPTION
**Pull request by teojgo**

### CODE REVIEW

File: easybuild/easyblocks/boostcray.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+##
```
Comment (victorusu) [inline easybuild/easyblocks/boostcray.py:1]: @teojgo, I do not understand the existence of this file...
Why is it here? None of this recipes use boost... There is no boost recipe...
Is it just to make it Python3 compatible?

Comment (teojgo) [reply easybuild/easyblocks/boostcray.py:1]: Yeah I need to add boost as well for 20.08, version 1.67

Comment (teojgo) [reply easybuild/easyblocks/boostcray.py:1]: @victorusu Boost is used by SerialBox

Comment (omlins) [reply easybuild/easyblocks/boostcray.py:1]: @victorusu , boostcray easyblock is needed, else COSMO does not build - tried several times to remove it, but it is required.

**Comment by victorusu** [comment]
@teojgo and @omlins, are we still deploying recipes using 20.08? 
Does this PR still makes sense?

**Comment by omlins** [comment]
As it is Theo's PR, he will know if he still needs that for some reason.

**Pull request closed without merging**